libs/coin-modules/coin-aptos/src/network/client.ts

```diff
@@ -25,15 +25,21 @@ import network from "@ledgerhq/live-network";
 import BigNumber from "bignumber.js";
 import isUndefined from "lodash/isUndefined";
 import { APTOS_ASSET_ID, DEFAULT_GAS, DEFAULT_GAS_PRICE, ESTIMATE_GAS_MUL } from "../constants";
-import { isTestnet } from "../bridge/logic";
+import { isTestnet, txsToApiOps } from "../bridge/logic";
```

Comment:
This file lists some rules we should follow regarding the structure and dependency relations on the coin-modules: libs/coin-modules/README.md

Reply:
done